draft-ietf-quic-transport.md

```diff
@@ -1627,7 +1627,7 @@ interface.  A client needs to start the connection process over if it migrates
 prior to completing the handshake.
 
 When a server receives an Initial packet with an address validation token, it
-SHOULD attempt to validate it, unless it has already completed address
+MUST attempt to validate it, unless it has already completed address
```

Comment:
MUST implies that it must fail if it cannot validate. During server cluster upgrades that might not be feasible. It could lead to consistent connection failures as long at the client has a token to try. It is in the servers interest to generally attempt validation so SHOULD seems reasonable to me.

Reply:
No that's not correct. The next sentence describes how to handle failures.

Reply:
Let's go back to 2119 definitions:  Are there situations in which a server might choose not to validate?  Will a failure to validate cause an interoperability problem?

Reply:
We also use MUST for security problems.

Reply:
There's an "unless" clause that captures the cases where a server won't validate, AFAICT. I'm ok with MUST here.

Also, the "it" is ambiguous. I would change the sentence to ".... MUST attempt to validate the token"